src/SFA.DAS.AANHub.Application.UnitTests/Apprentices/Commands/PatchApprenticeMember/PatchApprenticeMemberCommandValidatorTestsForName.cs

```diff
@@ -0,0 +1,95 @@
+﻿using FluentValidation.TestHelper;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Apprentices.Commands.PatchApprenticeMember;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.AANHub.Domain.Models;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Apprentices.Commands.PatchApprenticeMember
+{
+    [TestFixture]
+    public class PatchApprenticeMemberCommandValidatorTestsForName
+    {
+        private readonly Mock<IMembersReadRepository> _memberReadRepository;
+        private readonly Mock<IApprenticesReadRepository> _apprenticesReadRepository;
+
+        public PatchApprenticeMemberCommandValidatorTestsForName()
+        {
+            _memberReadRepository = new Mock<IMembersReadRepository>();
+            _apprenticesReadRepository = new Mock<IApprenticesReadRepository>();
+        }
+
+        private const string Replace = "replace";
+        private const string Name = "Name";
+
+        [TestCase("lorem epsum", true)]
+        [TestCase("", false)]
```

Comment:
I'd add a whitespace and null test too maybe?
```suggestion
        [TestCase(" ", false)]
```

Reply:
Applied for both name and email